dlp/noxfile.py

```diff
@@ -118,6 +118,17 @@ def system(session):
     session.install("-e", "../test_utils/")
     session.install("-e", ".")
 
+    # Additional setup for VPCSC system tests
+    if os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC") != "true":
+        # Unset PROJECT_ID, since VPCSC system tests expect this to be a project
```

Comment:
If I understand, `PROJECT_ID` is being used as a truthy value to control tests.

Why not use `GOOGLE_CLOUD_TESTS_IN_VPCSC` as that boolean?